src/shared/styles/components/top-bar.css

```
@@ -260,39 +273,38 @@
   min-width: 0;
-}
 
-.user-avatar {
-  width: 32px;
-  height: 32px;
-  border-radius: 50%;
-  object-fit: cover;
-  border: 1px solid rgba(158, 158, 158, 0.25);
-  flex-shrink: 0;
-}
+  overflow: hidden;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)